test/livebook/file_system/s3_test.exs

```diff
@@ -1,42 +1,25 @@
 defmodule Livebook.FileSystem.S3Test do
-  use ExUnit.Case, async: true
+  use Livebook.DataCase, async: true
 
   alias Livebook.FileSystem
   alias Livebook.FileSystem.S3
 
   setup do
     bypass = Bypass.open()
-    {:ok, bypass: bypass}
-  end
-
-  describe "new/3" do
-    test "trims trailing slash in bucket URL" do
-      assert %{bucket_url: "https://example.com/mybucket"} =
-               S3.new("https://example.com/mybucket/", "key", "secret")
-    end
```

Comment:
Any reason to drop this?

Reply:
Was in favor of `load/2`, but I'll rollback this

Reply:
We still use new for adding the file system, at least currently, so that's still the primary way to build it :)